src/main/java/com/vinylteam/vinyl/web/handler/ApiExceptionHandler.java

```diff
@@ -33,13 +34,12 @@
     public ResponseEntity<Object> handleServerExceptionApiExceptionHandler(ServerException exception) {
         return ResponseEntity.status(INTERNAL_SERVER_ERROR).body(new ApiExceptionDto(exception.getMessage()));
     }
-//FIXME: Fix situation with exceptions in a smart way.
-/*    @ExceptionHandler(RuntimeException.class)
-    public ModelAndView handleUserServiceException(RuntimeException exception) {
-        ModelAndView modelAndView = new ModelAndView("editProfile");
-        modelAndView.setStatus(BAD_REQUEST);
-        modelAndView.addObject("message", exception.getMessage());
-        return modelAndView;
-    }*/
+
+    @ExceptionHandler(value = {NotFoundException.class})
+    public ResponseEntity<Object> handleNotFoundExceptionApiExceptionHandler(NotFoundException exception) {
+        ApiExceptionDto apiExceptionDto = new ApiExceptionDto(
```

Comment:
return ResponseEntity.status(NOT_FOUND).body(new ApiExceptionDto(exception.getMessage()));
like it is in other methods of this class